sk0-front/src/app/auth/login/login.page.ts

```diff
@@ -0,0 +1,54 @@
+import { Component } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import {
+  FormsModule,
+  FormControl,
+  FormGroup,
+  ReactiveFormsModule,
+  Validators,
+} from '@angular/forms';
+import { IonicModule, NavController } from '@ionic/angular';
+import { AuthService } from '../auth.service';
+import { RouterModule } from '@angular/router';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.page.html',
+  styleUrls: ['./login.page.scss'],
+  standalone: true,
+  imports: [
+    IonicModule,
+    CommonModule,
+    FormsModule,
+    ReactiveFormsModule,
+    RouterModule,
+  ],
+})
+export class LoginPage {
+  loginForm = new FormGroup({
+    email: new FormControl('', []),
+    password: new FormControl('', []),
+  });
+
+  constructor(
+    private authService: AuthService,
+    private navCtrl: NavController
+  ) {}
+
+  async onLogin() {
+    const email = this.loginForm.get('email')!.value as string;
+    const password = this.loginForm.get('password')!.value as string;
+
+    this.authService.login({ email, password });
+
+    if (!this.authService.isUserLoggedIn) {
+      return;
+    }
+
+    this.navCtrl.navigateForward('/home');
```

Comment:
This should later be achieved by an auth guard rather later.

Reply:
sure, it will be a task in the near future, it is already in the backlog for the FE and BE as well. made just for testing purposes.